fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -575,6 +581,12 @@ protected void updateSchema() {
             oauthSchemaGenerator.buildOAuthSchema(pdm);
         }
 
+        // Build/update the Liberty JBatch related tables
```

Comment:
Do we assume the javabatch tables are in the same database as fhir data but with different schema?

Reply:
that's right.  Same database, same schema.

If necessary, I can add a separate command to deploy the Batch tables

Reply:
Thoughts?

Reply:
so, drop schema will keep the batch tables but only remove the fhirdata tables ... 
I think because we support different databases in the same fhir server, e.g, in CDT env, we configured performance## tenants using fhir-db2-perf1 database, and the others use databases of the other DB2 instances.   so, maybe it's better to use different job database for javabatch, because it's actually shared/used for export/import from/to all the other databases...  in CDT env, we used the same database as fhirdata, I think maybe this is not the best approach.  In my local, I always configure separated database for javabatch job repo...

Reply:
I think we'd better to separate the job repo.  but this is not a block for 4.2.2. if necessary, we can just create a issue for future enhancement.

Reply:
I've added --batch-only so you can deploy to a different db.  
It'll still install in the original main db, I don't think this is bad.

At least it can go into a separate db if needed (without the overhead of the rest of the tables)

Reply:
haha, just read this one and at least i understand the motivation for the `--batch-only` flag now